torchbenchmark/util/framework/huggingface/model_factory.py

```diff
@@ -59,6 +61,18 @@ def __init__(self, name, test, device, jit=False, batch_size=None, extra_args=[]
             input_ids = torch.randint(0, config.vocab_size, (self.batch_size, self.max_length)).to(device)
             decoder_ids = torch.randint(0, config.vocab_size, (self.batch_size, self.max_length)).to(device)
             self.example_inputs = {'input_ids': input_ids, 'labels': decoder_ids}
+
+            nbuckets = 8
```

Comment:
Can we place this in `get_dynamic_shapes_module()`? My concern is that when we testing non-dynamic shapes with large batch size, it will allocate large chunk of input data which occupies GPU memory.

We would like to only allocate GPU memory when it is needed.

Reply:
yea, that seems fair. I think i wanted to do it in parallel to the original way, and one concern is keeping the tensors the same if called 2x, but i could still do it inside get_dynamic_shapes_module and store/cache it on the self obj when it is created.